src/Tools/ExternalAccess/FSharp/Internal/Completion/FSharpInternalCommonCompletionProvider.cs

```diff
@@ -34,5 +35,11 @@ internal override bool IsInsertionTrigger(SourceText text, int insertedCharacter
         {
             return _provider.IsInsertionTrigger(text, insertedCharacterPosition, options);
         }
+
+        /// <summary>
+        /// Pulled from https://github.com/dotnet/fsharp/blob/master/vsintegration/src/FSharp.Editor/Completion/FileSystemCompletion.fs#L140
+        /// but this is incredibly fragile.
+        /// </summary>
+        internal override ImmutableHashSet<char> PossibleTriggerCharacters => ImmutableHashSet.Create('"', '\\', ',', '/');
```

Comment:
I think `PossibleTriggerCharacters` needs to be added to `IFSharpCommonCompletionProvider` as well, and this here would just return `_provider.PossibleTriggerCharacters`, this way F# can own keep the implementation in their repo. 
Also, since F# and TS need to implement their own LSP server, they would need this as well, we should discuss with them to make sure the API works for everyone.